velox/tool/trace/tests/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_executable(velox_tool_trace_test TableWriterReplayerTest.cpp)
+add_executable(velox_tool_trace_test TableWriterReplayerTest.cpp PartitionedOutputReplayerTest.cpp)
```

Comment:
Should we add a `Main.cpp` and put the main method in it as there will be more and more replayed tests?

Reply:
I'll remove the main() method. We don't need that in unit test.